analysis/log/make.log

```diff
@@ -1,23 +1,27 @@
 --------------------------------------------------------------------------------
-Makelog started: 2024-09-03 14:59:20
+Makelog started: 2024-09-03 18:52:21
 Working directory: /Users/zlxcindy/template_archive/analysis
```

Comment:
From the log file, it seems like you are pushing from your local end, not Sherlock?

Reply:
Thanks for pointing this out, I fixed it. Please check the [make.log](https://github.com/linxicindyzeng/template_archive/blob/5263895e3e14fd3e343bf2128856693ace16e426/analysis/log/make.log) file now. @ShiqiYang2022

Reply:
I also updated the PR detail with new links.